sensor_height: 0.8015
v_sample_num: 3
angle_width: 2.2
intensity_threshold: 200

max_slope: 0.1
max_height: 0.2

##
# max_vel_x: 0.5
# max_vel_theta: 1.0
# direction_sample: 15
# local_goal_range: 1.0

# w_lane: 1
# w_obs: 1
